packages/client/src/comp/Navbar.tsx

```diff
@@ -331,36 +334,60 @@ export const Navbar = () => {
         <i className="fa fa-globe"/>
         <i className="fa fa-chevron-down"/>
       </NavToggleButton>
-      {/* Top 5 non-English languages, in order https://en.wikipedia.org/wiki/Languages_of_the_United_States#Most_common_languages */}
+      {/* Top 8 non-English languages, in order https://en.wikipedia.org/wiki/Languages_of_the_United_States#Most_common_languages */}
       <div className='picker' onClick={() => toggleExpanded('translation')}>
-        <a href={`https://translate.google.com/translate?hl=&sl=en&tl=es&u=${url}`}>
+        <a
```

Comment:
Let's make them open a new target (not just embed ones).

Reply:
Just one small change, once this passes testing, feel free to merge yourself @ludanin